### DISCUSSION TITLE
How to delete my discussion ?

### DISCUSSION THREAD

**Discussion by zwelhtetyan** [community/community#52581] [Discussions] [answered]

**Comment by sahana-github** [comment, marked as answer]
Hi @zwelhtetyan 
Go to the discussion you want to delete.
Click on the three-dot icon on the top right corner of the discussion.
Select "Delete discussion" from the dropdown menu.
Confirm the deletion by clicking on the "Delete discussion" button.

**Reply by zwelhtetyan** [reply]
HI @sahana-github, there is no three-dot in my case as below.

![Screenshot from 2023-04-13 18-49-55](https://user-images.githubusercontent.com/89287841/231756701-bdab254c-b9ce-4e4f-b00c-ac3cffd5fd7e.png)

**Comment by sahana-github** [comment]
@zwelhtetyan 

![Untitled](https://user-images.githubusercontent.com/82923085/231759249-0aed10bf-620d-4b04-84ea-5c07a29ebbd8.png)

**Reply by roke-julian-lockhart** [reply]
@sahana-github, I don't have that option at [`Murmele/Gittyup/discussions/520#discussion-5102492`](https://github.com/Murmele/Gittyup/discussions/520#discussion-5102492):

![image](https://github.com/user-attachments/assets/3d5440be-ae5e-4a70-9109-d4dacb57e9a3)

Nor do I have it at [`orgs/community/discussions/193795`](https://github.com/orgs/community/discussions/193795#discussion-9943766), wherein it's incorrectly-tagged.
